cli/bin/internal_command_update.dart

```diff
@@ -27,5 +27,7 @@ class InternalCommandUpdate extends InternalCommand {
         .announce("Pulling new changes.")
         .runSync()
         .printNotEmptyResultFields();
+    context.printToConsole(
```

Comment:
Let's delete all the code in the function except this newly added one. 

Also, in line # 7 of this file, update command to become hidden.

```
InternalCommandUpdate() : super("update", "Updates to the latest version.", type: InternalCommandType.hidden);
```